data/binding/convert_test.go

```diff
@@ -44,7 +44,7 @@ func TestBoolToStringWithFormat(t *testing.T) {
 	assert.Nil(t, err)
 	v, err = s.Get()
 	assert.Nil(t, err)
-	assert.Equal(t, "truely", v)
```

Comment:
Sadly this line (and the one below) is not going to work, as the test is for "true"+"ly".
Feel free to make a slightly larger change that does not mark it as a typo, but correcting the spelling is what has caused the build failures.

Reply:
Thanks.